docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -9,80 +9,151 @@ track: "USERS › APPLICATION DEVELOPER › FOUNDATIONAL"
-* Optionally, learn more about providing external access:
-    * [More about Deployments](/docs/concepts/workloads/controllers/deployment/).
-    * [More about Services](/docs/concepts/services-networking/service/).
-    * [Learn more about Ingress](/docs/concepts/services-networking/ingress/).
+   Minikube includes a Docker daemon, but to create and push {% glossary_tooltip text="containers" term_id="container" %} as part of your development workflow, you'll want an independent Docker instance. Note that version 1.12 is recommended for full compatibility with Kubernetes, but a few other versions are tested and known to work.
 
-You'll also want to think about storage. Kubernetes provides different types of storage for different storage needs:
 
-* A Volume lets you define storage for your cluster that is tied to the lifecycle of a Pod. It is therefore more persistent than container storage. Learn [how to configure volume storage](/docs/tasks/configure-pod-container/configure-volume-storage/), or [read more about about volume storage](/docs/concepts/storage/volumes/).
-* A PersistentVolume and PersistentVolumeClaim let you define storage at the cluster level. Typically a cluster administrator defines the PersistentVolume objects for the cluster, and cluster users (application developers, you) define the PersistentVolumeClaim objects that your application requires. Learn [how to set up persistent storage for your cluster](/docs/tasks/configure-pod-container/configure-persistent-volume-storage/) or [read more about persistent volumes](/docs/concepts/storage/persistent-volumes/).
+You can get basic information about your cluster with the command `kubectl get nodes`. To get a good idea of what's really going on, however, you need to deploy an application to your cluster. This is covered in the next major section.
+
+## Deploy an application
+
+#### A Basic Workload
+
+The following examples demonstrate how to deploy basic apps to Kubernetes:
+  * [Deploy a simple, stateless nginx server](/docs/tasks/run-application/run-stateless-application-deployment/){:target="_blank"}. Nginx is often used as the web server to help provide ingress from outside your cluster (in other words, external requests).
+
+  * [Deploy a stateful, externally accessible MySQL database](/docs/tasks/run-application/run-single-instance-stateful-application/){:target="_blank"}. This introduces the concept of storage.
+
+Through these deployment tasks, you'll gain familiarity with the following concepts:
+* Fundamentals
+
+  * **Kubernetes manifests** - These configuration files describe the desired state of your application and related resources as Kubernetes should maintain them. They are written in YAML or JSON. (See the [example from the nginx app](/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment))
+
```

Comment:
I think it's good to maintain a distinction between manifest and file. A single file could contain several resource manifests. Maybe this:

**Configuration files** - These files describe ...